### PR TITLE
feat: navbar, footer and submenu added

### PR DESCRIPTION
**Pull request by partha8**
## Things to check
- [src/components/Footer/Footer.js](src/components/Footer/Footer.js)
- [src/components/Navbar/Navbar.js](src/components/Navbar/Navbar.js)
- [src/components/Submenu/Submenu.js](src/components/Submenu/Submenu.js)

## Things to avoid
- The CSS files

### Disclaimer
I know the categories in the navbar and submenu can be mapped over, I'll work on it when soon. Thank you.

### CODE REVIEW

**Review by VLeads — APPROVED** [review]
Job nicely done as of now. Go ahead !!

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import "./navbar.css";
+import { FaUserAlt, FaShoppingBag, FaHeart, FaSearch } from "react-icons/fa";
+import { GiHamburgerMenu } from "react-icons/gi";
```
Comment (ShekharDhangar) [inline src/components/Navbar/Navbar.js:4]: Try to import all icons in one file and export from that file, so import lines will reduce.

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                wishlist.length ? "badge icon-badge" : "hide-badge"
+              }`}
+            >
+              {wishlist.length}
+            </span>
+          </Link>
+          <FaSearch />
+        </div>
+      </div>
+      <ul id="links-container" className="links-container">
```
Comment (ShekharDhangar) [inline src/components/Navbar/Navbar.js]: why need id and classname and both same?
Is that intentional or mistake?

File: src/context/StateProvider.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      }}
+    >
+      {children}
+    </StateContext.Provider>
+  );
+};
+
+const useStateContext = () => {
+  return useContext(StateContext);
+};
```
Comment (ShekharDhangar) [inline src/context/StateProvider.js:39]: you could remove curly brackets and return keyword and do implicit return

**Review by ShekharDhangar — APPROVED** [review]
Try to do the suggested changes, rest looks good.

File: src/components/Navbar/Navbar.js (hunk 2 above)
Comment (partha8) [reply src/components/Navbar/Navbar.js]: It's a mistake, as I directly copy-pasted the HTML code from the ecom screens.

File: src/context/StateProvider.js (hunk 3 above)
Comment (partha8) [reply src/context/StateProvider.js:39]: It's a force of habit.

**Pull request merged**